rdrf/rdrf/templates/debug/outbox.html

```diff
@@ -0,0 +1,31 @@
+{% extends "rdrf_cdes/base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+  <h1>Mail Outbox</h1>
+  {% if mail_messages %}
+    <table class="table table-striped" id="id_messages">
```

Comment:
nit: Long messages make the table columns spill outside the page. Could either make the text wrap, or the individual cells have an `overflow` css property
![image](https://user-images.githubusercontent.com/17246877/181170840-12a3e891-ce33-4870-83a8-4e8ca63e29a0.png)